model/src/main/scala/uk/gov/ons/addressIndex/model/db/index/HybridIndex.scala

```diff
@@ -0,0 +1,64 @@
+package uk.gov.ons.addressIndex.model.db.index
+
+import com.sksamuel.elastic4s.{HitAs, RichSearchHit}
+import play.api.libs.json.{JsValue, Json}
+import uk.gov.ons.addressIndex.model.db.ElasticIndex
+
+/**
+  * PAF Address DTO
+  */
+case class HybridIndex(
+  uprn: String,
+  lpi: JsValue,
+  paf: JsValue
+)
+
+object HybridIndex extends ElasticIndex[HybridIndex] {
+
+  val name: String = "hybrid"
+
+  implicit lazy val fmt = Json.format[HybridIndex]
```

Comment:
Ah, most of this file is legacy, managed to strip it right back to minimal stuff